packages/network-controller/tests/provider-api-tests/block-param.ts

```diff
@@ -70,7 +91,8 @@ export const testsForRpcMethodSupportingBlockParam = (
         continue;
       }
 
-      it(`does not reuse the result of a previous request if parameter at index "${paramIndex}" differs`, async () => {// eslint-disable-line
+      // eslint-disable-next-line no-loop-func
```

Comment:
What's this override for?

Reply:
same as what was there, just being more specific now (no-loop-func being the lint rule that gets upset about this file)